sdk/search/azure-search/src/main/java/com/azure/search/implementation/util/DocumentResponseConversions.java

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.search.implementation.util;
+
+import com.azure.core.exception.HttpResponseException;
+import com.azure.core.exception.ResourceNotFoundException;
+import com.azure.search.SearchDocument;
+
+/**
+ * Utility class for Document Response conversions.
+ */
+public final class DocumentResponseConversions {
+
+    /*
+     * Represents the OData context field.
+     */
+    private static final String ODATA_CONTEXT = "@odata.context";
```

Comment:
Did you make the change to emit `application/json;odata.metadata=none` for the `Accept` header of each request from `SearchIndexClient`? If so, you shouldn't need this anymore.

Reply:
Removed

Reply:
I took over Alan's PR as he is OOF, I found it is not possible to checkin his repo recently. As this is internal use class. We can leave the change for next release.